support-models/src/main/scala/com/gu/support/catalog/Product.scala

```diff
@@ -17,12 +17,14 @@ sealed trait Product {
     environment: TouchPointEnvironment,
     billingPeriod: BillingPeriod,
     fulfilmentOptions: FulfilmentOptions,
-    productOptions: ProductOptions
+    productOptions: ProductOptions,
+    readerType: ReaderType = Direct,
   ): Option[ProductRatePlan[Product]] =
     ratePlans(environment).find(prp =>
       prp.billingPeriod == billingPeriod &&
         prp.fulfilmentOptions == fulfilmentOptions &&
-        prp.productOptions == productOptions
+        prp.productOptions == productOptions &&
+        prp.readerType == readerType
```

Comment:
was this a bug before? the missing readerType check.

Reply:
Only a theoretical one as this method was never used in the gifting or corporate code I don't think